src/contact/Contact.jsx

```diff
@@ -49,9 +140,19 @@ export function Contact() {
 			/>
 		</form>
 		<div>
-			<Button variant="contained" color="primary" onClick={() => sendEmail()}>
+			<Button id="count-up" variant="contained" color="primary" onClick={() => validateInputs()} disabled={disabledButton}>
```

Comment:
se não estou enganado, você pode só passar a função aqui. `onClick={validateInputs()}`

Reply:
Usar ids pode se tornar um problema. 
Reuni alguns arquivos que podem ajudar:
* https://www.javascriptstuff.com/use-refs-not-ids/
* https://csswizardry.com/2011/09/when-using-ids-can-be-a-pain-in-the-class/ (Esse da o exemplo com CSS, mas é valido para outras coisas também, como react)